src/login.html

```diff
@@ -45,13 +45,13 @@
                         </form>
                     </div>
                     <div class="loggedin" id="loggedin">
-                        <button class="btn btn-danger" name="Logout" onClick="logout(this)" id="logout">Logout</button>
```

Comment:
Also, any reason for changing this line?

Reply:
Content Security Policy does not allow inline javascript. So you have to put your javascript in a .js file and include it in your HTML.